apps/storybook.namekit.io/stories/Namekit/Input.stories.tsx

```diff
@@ -112,6 +112,17 @@ export const WithSlotRight: Story = {
   },
 };
 
+export const WithError: Story = {
+  args: {
+    variant: "primary",
+    inputSize: "medium",
+    placeholder: "Say something",
+    slotPosition: "right",
+    error: "This is an error",
```

Comment:
```suggestion
    error: "Email is required",
```

One of our goals for these stories is to define them to be less academic and theoretical and instead to make them more practical real life situations.